cf/api/services_test.go

```diff
@@ -203,7 +203,7 @@ var _ = Describe("Services Repo", func() {
 				Response: testnet.TestResponse{Status: http.StatusCreated},
 			}))
 
-			err := repo.CreateServiceInstance("instance-name", "plan-guid", nil)
+			err := repo.CreateServiceInstance("instance-name", "plan-guid", nil, nil)
```

Comment:
I kind of hate seeing these `nil`s all over the place when we could use empty string array...  But eh, that's just me.

Though hopefully we are testing for case when tags is empty array vs nil...